### DISCUSSION TITLE
Shows a not found error when loading a route directly

### DISCUSSION THREAD

**Discussion by Judge-Paul** [gilbarbara/react-joyride#986] [General]
When you load a route directly instead of navigating in the app it throws a 404 Error 
For example going to this route directly https://react-joyride.com/multi-route will throw an error 
![Screenshot from 2024-01-06 15-55-04](https://github.com/gilbarbara/react-joyride/assets/110723341/4d6065bb-fe51-423d-a4f7-18888e3ba21b)

## To Reproduce

Enter the site URL followed by any route to see this not found error

**Comment by Judge-Paul** [comment]
@gilbarbara Hey, you made this a discussion 👀

**Reply by gilbarbara** [reply]
Yes, it's no a bug in the code

**Reply by Judge-Paul** [reply]
Oh, okay
What platform did you use to host it maybe I can help

**Reply by gilbarbara** [reply]
Fixed.
Thanks
